cookbook/localization/index.rst

```diff
@@ -0,0 +1,44 @@
+How to Use the Localizers
+=========================
+
+Localizers are used to convert a localized attribute to the standard format.
+Eq: a decimal with a comma separator have to be convert with a dot separator before insert in database.
+
+How to Register a New Localizer
+-------------------------------
+
+The Akeneo PIM comes with a number of localizers.
+It also comes with a flexible way to define your own localizer.
```

Comment:
"The Akeneo PIM comes with several native localizers, but provides also a flexible way to define your own localizers." ?